### PR TITLE
Move importproof RPC endpoint into new tapdev RPC server

### PR DESCRIPTION
**Pull request by ffranr**
This PR fixes: https://github.com/lightninglabs/taproot-assets/issues/395

### CODE REVIEW

File: taprpc/tapdevrpc/tapdev.swagger.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
+{
```
Comment (guggero) [inline taprpc/tapdevrpc/tapdev.swagger.json:1]: I think we should squash this commit with the previous one, otherwise the first one won't compile?

**Review by guggero — COMMENTED** [review]
Nice work! Have a couple of small comments, should be pretty easy to address.

File: cmd/tapcli/tapdevrpc_active.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import (
+	"fmt"
+
+	"github.com/lightninglabs/taproot-assets/taprpc/tapdevrpc"
+	"github.com/lightningnetwork/lnd/lncfg"
+	"github.com/urfave/cli"
+)
+
+// devCommands will return the set of commands to enable for tapdevrpc builds.
+func devCommands() []cli.Command {
```
[the remark this answers is not in the document]
Comment (ffranr) [reply cmd/tapcli/tapdevrpc_active.go:15]: It's a function in lnd, but a list in tap. I will change it into a list.

**Review by positiveblue — APPROVED** [review]
LGTM :+1:

Not sure if all the commits building is a must, if so we may have to rearange some of the changes to be sure that we do not import things that do not exist.

**Review by guggero — APPROVED** [review]

**Pull request merged**